src/explore-education-statistics-frontend/src/modules/data-catalogue/DataCataloguePage.tsx

```diff
@@ -144,6 +154,45 @@ const DataCataloguePage: NextPage<Props> = ({
     );
   };
 
+  const handleSupersededWarningSubmit = () => {
+    const type = themes
+      .flatMap(theme => theme.topics)
+      .flatMap(topic => topic.publications)
+      .find(
+        publication =>
+          publication.id === state.query.publication?.supersededById,
+      )?.type;
+
+    updateState(draft => {
+      draft.query.publication = {
+        id: state.query.publication?.supersededById as string,
+        slug: state.query.publication?.supersededBySlug as string,
+        title: state.query.publication?.supersededByTitle as string,
+        isSuperseded: false,
+        // TODO: Remove 'type' field (see EES-3839) as it is unused on new find stats page
+        // (old-find stats page previously used 'type' to group publications by their type)
+        type: type as PublicationType,
```

Comment:
Hopefully this is fixed by https://github.com/dfe-analytical-services/explore-education-statistics/pull/3756!